cores/rp2040/SerialUART.cpp

```diff
@@ -365,6 +366,23 @@ SerialUART::operator bool() {
     return _running;
 }
 
+bool SerialUART::getBreakReceived() {
+    CoreMutex m(&_fifoMutex);
```

Comment:
Drop the `CoreMutex` and adjust the condition only on `_running`, please.  If we lock here the IRQ stuff will be blocked from running since the mutex will be taken,

Reply:
OK